pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
   "packageurl-python",
   "packaging",
   "pydantic",
+  "pypi-simple",
```

Comment:
I noticed that `requirements-extras.txt` wasn't updated - `make pip-compile` should fix that